string ParameterType
float64 I_R
float64 C_R
float64 FK_R
float64 MgL_R
float64 PID_Gain_R
float64 N_RH
float64 I_L
float64 C_L
float64 FK_L
float64 MgL_L
float64 PID_Gain_L
float64 N_LH
float64 D_RH
float64 D_LH
int32 P_RH
int32 P_LH
float64 Delta_Theta_Start
float64 Delta_Theta_End
float64 Epsilon